docker/Dockerfile-datascript

```diff
@@ -0,0 +1,17 @@
+# Build client images for mysql and postgres to support datascripts
+# Use alpine with tag 20230329 is corresponding to "edge" tag (latest release to date is 3.18) as of 20230625
+FROM docker.io/alpine:edge as dist
+# ARG APK_MIRROR
+
+# install tools via apk
+ENV APK_MIRROR=${APK_MIRROR}
+RUN if [ -n "${APK_MIRROR}" ]; then sed -i "s/dl-cdn.alpinelinux.org/${APK_MIRROR}/g" /etc/apk/repositories; fi
+
+RUN apk add --no-cache jq --allow-untrusted
+
+RUN apk add --no-cache postgresql-client --allow-untrusted
+
+RUN apk add --no-cache mysql-client mariadb-connector-c --allow-untrusted
+RUN rm -rf /var/cache/apk/*
```

Comment:
Cause the image itself is about 100MB with mysql-client and pg-client. But it is not a necessary to most works. So I think build it soley would be better (otherwise the tools image will be a big-fat image)

Reply:
ack